docs/cloud-context/namespaces-create.md

```diff
@@ -29,7 +29,7 @@ To create a Namespace in Temporal Cloud, gather the following information:
 ### Create a Namespace using Temporal Cloud UI
 
 1. Gather the information listed earlier in [Information needed to create a Namespace](#information-needed-to-create-a-namespace).
-1. In a browser, go to [https://cloud.temporal.io](https://cloud.temporal.io) and log in.
+1. Go to to Temporal Cloud UI and log in.
```

Comment:
De-dupe "to to". (Doing reviews on a phone sucks.)

Reply:
Good catch!